scan/src/components/oracle-script/OracleScriptBridgeCode.re

```diff
@@ -0,0 +1,213 @@
+      {
+          Borsh.Data memory data = Borsh.from(_data);
+          result.symbol = string(data.decodeBytes());
+          result.multiplier = data.decodeU64();
+          result.what = data.decodeU8();
+      }
+    }|j};
+
+  let (targetPlatform, setTargetPlatform) = React.useState(_ => "Ethereum");
+  let (language, setLanguage) = React.useState(_ => "Solidity");
+  <div className=Styles.tableWrapper>
+    <>
+      <VSpacing size={`px(10)} />
+      <Row>
+        <HSpacing size={`px(15)} />
+        <Col>
+          <div> <Text value="Target Platform" /> </div>
+          <VSpacing size={`px(5)} />
+          <div className=Styles.selectWrapper>
+            <TargetPlatformIcon icon=targetPlatform />
+            <select
+              className=Styles.selectContent
+              onChange={event => {
+                let newValue = ReactEvent.Form.target(event)##value;
+                setTargetPlatform(_ => newValue);
+                switch (newValue) {
+                | "Ethereum" => setLanguage(_ => "Solidity")
+                | "Cosmos IBC" => setLanguage(_ => "Go")
+                | "Kadena" => setLanguage(_ => "PACT")
+                | _ => setLanguage(_ => "Solidity")
+                };
+              }}>
+              {[|"Ethereum", "Cosmos IBC", "Kadena"|]
+               ->Belt_Array.map(symbol => <option value=symbol> {symbol |> React.string} </option>)
+               |> React.array}
+            </select>
+          </div>
+        </Col>
+        <HSpacing size={`px(370)} />
+        <Col>
+          <div> <Text value="Language" /> </div>
+          <div className=Styles.selectWrapper>
+            <LanguageIcon icon=language />
+            <select
+              className=Styles.selectContent
+              onChange={event => {
+                let newValue = ReactEvent.Form.target(event)##value;
+                setLanguage(_ => newValue);
+              }}>
+              {switch (targetPlatform) {
+               | "Ethereum" =>
+                 [|"Solidity", "Vyper"|]
+                 ->Belt_Array.map(symbol =>
+                     <option value=symbol> {symbol |> React.string} </option>
+                   )
+                 |> React.array
+               | "Cosmos IBC" => <option value="Go"> {"Go" |> React.string} </option>
+               | "Kadena" => <option value="PACT"> {"PACT" |> React.string} </option>
+               | _ =>
+                 [|"Solidity", "Vyper"|]
+                 ->Belt_Array.map(symbol =>
+                     <option value=symbol> {symbol |> React.string} </option>
+                   )
+                 |> React.array
+               }}
+            </select>
+          </div>
+          <VSpacing size={`px(5)} />
+        </Col>
+      </Row>
+      <VSpacing size={`px(35)} />
+      <div className=Styles.tableLowerContainer>
+        <div className=Styles.vFlex>
+          <Text value="Description" size=Text.Lg color=Colors.gray7 weight=Text.Medium />
```

Comment:
Let's use the default font-weight and add letter-spacing by add prop called `spacing={Text.Em(0.03}`.